hathor/transaction/genesis.py

```diff
@@ -41,3 +46,40 @@ def get_representation_for_all_genesis(settings: HathorSettings) -> bytes:
 def is_genesis(hash_bytes: bytes, *, settings: HathorSettings) -> bool:
     """Check whether hash is from a genesis transaction."""
     return hash_bytes in get_all_genesis_hashes(settings)
+
+
+def generate_new_genesis(
+    *,
+    tokens: int,
+    address: str,
+    block_timestamp: int,
+    min_block_weight: float,
+    min_tx_weight: float,
+) -> tuple[Block, Transaction, Transaction]:
+    """Crate new genesis block and transactions."""
```

Comment:
typo: Create ...

Maybe add that it is typically used to create new side DAGs.

Reply:
Done in https://github.com/HathorNetwork/hathor-core/pull/1069/commits/a5db4fc4a5c1a0e7058df7bfd03dde3d9e3b027e

Reply:
Thanks!